guides-and-tutorials/running-a-signer/best-practices-to-run-a-signer.md

```diff
@@ -10,15 +10,6 @@ The following best practices suggest how to create a resilient setup for running
 tl;dr: avoid single point of failures, introduce redundancy, monitor things.
 {% endhint %}
 
-### Run redundant Signer instances
```

Comment:
Add a comment in Monitor your signer: "Ensure that only one instance is using the same private key"

Reply:
This doesn't fit in the "monitor" section, but I agree we might need to make it more explicit.